book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] As required for the project to correctly respect  the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. kindly ensure that: `Getters` and `Setters` for `@rentals` are present, `@rentals` should also be present in your constructor method as instance variable, and you should also have a method for adding rentals where the book itself is sent as a parameter to create the new rental achieving the 'has-many' association. It should be similar to this:
```ruby
class Book
  attr_accessor :title, :author, :rentals
  .
  .
  def initialize(title, author)
    .
    .
    @rentals = []
  end
  
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```

Reply:
Hello @tresorsawasawa, Thanks for the review but I think the requirements stated what should be done clearly and if am not doing it right then the project requirement is not write. This is the requirement and i can’t see the request to create a method `Create a class Book with the following:
@title and @author instance variables, should be initialized in the constructor.
Setters and getters for instance variables (remember about attr_accessor).`.  Anyway, I will go ahead with your suggestions.

Reply:
Hi @ctokoli 

As the `@rentals` depends on your relationship that why I mentioned that, you can even check all the requirements
![Screenshot from 2023-09-13 14-01-07](https://github.com/ctokoli/OOP-school-library/assets/83042742/46f15f19-c3e9-4c29-8ec9-b278f9137e81)